Documentation/project-docs/pull-request-validation.md

```diff
@@ -0,0 +1,16 @@
+In the .NET Platform repository we agreed on the following policies before a Pull Request can be merged:
+
+1. Every PR needs to be **approved** by a team member with write access to the repository.
+2. Unless changes aren't protected by **Continuous Integration** (CI), i.e. Markdown edits, CI must have run recently.
+3. The **corefx-ci badge** which indicates that all legs run successfully must be **green**, except an [unrelated test failure](#unrelated-test-failure) happened.
+
+These rules apply to both community and team members.
+
+## Unrelated test failure
+In case Continuous Integration indicates test failures which are **highly unlikely** (in doubt, hit retry) to be caused by changes in the Pull Request, the following actions should be taken:
```

Comment:
> "when in doubt, retry"

Developers need to be thoughtful about retries.  There are plenty of bugs, e.g. race conditions, where a failure might highlight a real problem and it won't manifest again on a retry.  We don't want to paper over such issues.

Reply:
Right, we don't want to have a habit of retrying before filing a bug and disabling a test.